doc/FAQ.md

```diff
@@ -65,6 +65,29 @@ only to HTTP requests.
 You can use [Let's Encrypt](https://letsencrypt.org/) to get an SSL certificate
 for free.
 
+## How do I access web services?
+
+code-server is capable of proxying to any port using either a subdomain or a
+subpath.
+
+### Sub-domains
+
+Set up a wildcard certificate for your domain and a wildcard DNS entry (or you
```

Comment:
We should also point out that you can manually set records for `3000.<domain>`, `8080.<domain>`, etc. Cloudflare doesn't support wildcard domains with TLS

Reply:
That's actually what I meant by the text in the parenthesis but I don't think it was very clear. What about something like:

> You will need a DNS entry that points to your server for each port you want to access. You can either set up a wildcard DNS entry for `*.<domain>` if your domain name registrar supports it or you can create one for every port you want to access (`3000.<domain>`, `8080.<domain>`, etc). 

> You should also set up TLS certificates for these subdomains, either using a wildcard certificate for `*.<domain>` or setting up individual certificates for each port.